### PR TITLE
Fix Netlify bug introduced by changing editorial workflow

### PR DESCRIPTION
**Pull request by jmdembe**
I am trying to fix a bug where if a user wants to create a new article, they are build fails and users are not able to submit new articles.

The new `netlify_sanbox.md` file _should_ make the test pass because there is a matching tile on the home page that will take you to the `netlify-sandbox` sandbox articles. I get false positives when testing locally, so I am going on what I read from the test and matching it to other categories.

### CODE REVIEW

File: admin/config.yml [hunk 1]
```diff
@@ -18,6 +18,7 @@ collections:
     name: help
     folder: _articles
     create: true
+    path: '{{category}}/{{slug}}'
```
Comment (zachmargolis) [inline admin/config.yml:21]: is this the UI path? or the file path?
all articles are in the same directory (`_articles`) and `/articles`

**Review by zachmargolis — COMMENTED** [review]
Not 100% sure I follow but that's because I am not an Netlify expert

File: admin/config.yml (hunk 1 above)
Comment (jmdembe) [reply admin/config.yml:21]: yikes, that is the file path. I was messing with some things, but didn't remove this.

**Review by zachmargolis — APPROVED** [review]
LGTM

**Pull request merged**